src/baseconverter.js

```diff
@@ -1,18 +1,30 @@
 'use strict';
 
+
 function normalize(digitArray) {
-  while (digitArray[0] === 0) {
-    digitArray.shift();
+  const length = digitArray.length;
```

Comment:
0.13 has completely changed api, so I can't imagine that RippleTrade will use it.
and most of my changes here went to OrderBook and AutobridgeCalculator, which, I believe, doesn't changed much in 0.13, so it will be easy port my changes to 0.13